src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -18,44 +18,11 @@ freshnessValidatedDate: never
 
-## Enable Kubernetes for log management [#enable-process]
-
-To forward your Kubernetes logs to New Relic with our plugin:
-
-1. [Install the New Relic Kubernetes integration](/install/kubernetes). This integration includes the Kubernetes plugin for logs.
-
-2. Optionally, you can further tune your installation in [the guided install](/install/kubernetes) using the numerous configuration options available in the [`newrelic-logging` repository](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging#configuration). However, we recommend the standard setup, as it's valid for most users.
-
-    <img
-        title="Guided install from the UI"
-        alt="Guided install from the UI"
-        src="/images/kubernetes_screenshot-crop_gather-log-data.webp"
-    />
-
-    <figcaption>
-        Go to <DNT>**[one.newrelic.com](https://one.newrelic.com/) > Integrations & Agents**</DNT> and click the <DNT>**Guided install**</DNT> tile. Then select <DNT>**Kubernetes**</DNT> and <DNT>**Guided**</DNT>.
-    </figcaption>
-
-    <Callout variant="important">
-
-    If you're [using a Kubernetes secret](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/values.yaml#L8-L25) to store the New Relic <InlinePopover type="licenseKey" />, the `newrelic-logging` chart defaults to sending logs to the US API endpoint. If the license key belongs to an EU or FedRAMP account, and a secret is used for key storage, you must update the endpoint setting with the appropriate value from the [API reference docs](/docs/logs/log-api/introduction-log-api/#endpoint). Here's an example of how to set this for EU accounts:
-
-    ```yml
-        newrelic-logging:
-            enabled: true
-            endpoint: https://log-api.eu.newrelic.com/log/v1
-    ```
-    </Callout>
-
-3. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
-
-<InstallFeedback />
-
 ## Fluent Bit output plugin [#fluent-bit]
 
-New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
+New Relic has a [Fluent Bit output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
 
-See [Fluent Bit plugin for log forwarding](/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding) for more details about it.
+See [Fluent Bit plugin for log forwarding](/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding) doc for more details about it.
 
 ### Additional metric details [#addition-metric-details]
```

Comment:
@nbaenam is all this information about metric data just applicable to the Helm chart version of the New Relic Fluent Bit Docker Container?